resources/dicts/events/injury/general/apprentice.json

```diff
@@ -317,7 +386,7 @@
             "Leaf-fall",
             "Leaf-bare"
         ],
-        "event_text": "m_c trained extra hard to impress their mentor, but were a bit too rough with their body and have the bruises to show it.",
+        "event_text": "m_c trained extra hard to impress their mentor, but were a bit too rough with their body, and have the bruises to show it.",
```

Comment:
i think should be "but was a bit too rough" instead of "but were a bit too rough"